plone/app/contenttypes/interfaces.py

```diff
@@ -13,8 +14,21 @@ class IPloneAppContenttypesLayer(Interface):
 
 
 class ICollection(Interface):
-    """Explicit marker interface for Collection
-    """
+
+    """Explicit marker interface for Collection."""
```

Comment:
When you add attributes and methods, it is no longer a marker interface. This also means that `alsoProvides(context, ICollection)` will not work. This may or may not be a problem.

There are two interfaces: the current one in `interfaces.py` and the behavior in `behavior/interfaces.py`. I am one of the persons who created this a few years ago. I'm not sure it still makes sense this way. I don't think I had created many behaviors before this. But all interfaces in the main `interfaces.py` are marker interfaces, so I think it is best to keep it that way.

But documentation is good. I suggest keeping the current marker interface without attributes and methods, and adding a line in the docstring that points to `behaviors/interfaces.py`.